internal/argtype.go

```diff
@@ -26,13 +26,21 @@ var (
 		Use:   "yo PROJECT_NAME INSTANCE_NAME DATABASE_NAME",
 		Short: "yo is a command-line tool to generate Go code for Google Cloud Spanner.",
 		Args: func(cmd *cobra.Command, args []string) error {
-			if len(args) != 3 {
+			if cmd.Flag("create-templates").Changed {
```

Comment:
xo does the same thing to run different commands by using flags. But it is hard to understand what flags are available for commands.
I think we will have another commands for the future, so we should introduce commands instead of magical handling flags.

Please use arguments instead of flags. At first, if the number of arguments is 1 and the argument is "create-template", run the command. Otherwise, keep the current behavior.  Then I will do some change next.

Reply:
@kazegusuri 

I want to confirm the method how to handle `if the number of arguments is 1`.
Do you intend adding a subcommand by using cobra function?
For examle below.

```go
// internal/create_templates.go
package internal

var createTemplatesCmd = &cobra.Command{
  ...
}

func init() {
  rootCmd.AddCommand(createTemplatesCmd)
}
```